src/shinyhunter/shiny_hunter_controller.py

```
@@ -49,41 +51,106 @@ def countdown(self, seconds: int):
-                
-                # Only increment count after successful encounter verification
-                self.increment_count()
-                self.log(f"Attempt #{self.count}")
-            else:
-                # Fallback to original method
+
+            if config.use_custom_sequence:
+                # --- Custom sequence mode ---
+                try:
+                    steps = self._load_custom_sequence_steps()
+                except (FileNotFoundError, json.JSONDecodeError, Exception) as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)